front/pages/w/[wId]/builder/data-sources/public-urls.tsx

```diff
@@ -241,28 +258,24 @@ export default function DataSourcesView({
 }
 
 function getTableColumns() {
-  // to please typescript
-  type Info = {
-    row: {
-      original: DataSourceType & {
-        icon: ComponentType;
-        usage: number;
-      };
-    };
-  };
   return [
     {
       header: "Name",
+      id: "name",
       accessorKey: "name",
       cell: (info: Info) => (
         <DataTable.Cell icon={info.row.original.icon}>
-          {info.row.original.name}
+          <span className="hidden sm:inline">{info.row.original.name}</span>
+          <span className="inline sm:hidden">
+            {truncate(info.row.original.name, 30, "...")}
+          </span>
```

Comment:
can't we use a simple text ellipsis in the table ?

Reply:
Problem is that `DataTable.Cell` width isn't based on the device (which should be the case but this can be done later). So we end up having columns that are as wide on sm devices as they are on large devices -> adding ellipsis doesn't do the trick here 😕